CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -545,12 +545,12 @@ msgstr ""
 
 #. 3.2->settings-schema.json->search-filesystem->tooltip
 #. 2.6->settings-schema.json->search-filesystem->tooltip
-msgid "Allows path entry in the menu search box."
+msgid "Allows path entry into the menu search box."
```

Comment:
This sounds wrong to me. This allows you to search for paths in (not into?) the search box?